README.md

```diff
@@ -120,13 +120,12 @@ Then point your browser to http://localhost:3000/.
 
 ## Local Testing
 
-To run a build that can be locally tested using any HTTP server, run
+To run a build that can be locally tested using any HTTP server:
```

Comment:
Does any HTTP server work with the new single-page app structure?

Reply:
Yup. I tested it with `live-server` and `python -m SimpleHTTPServer`.